docs/_includes/projects/docs/expressions/custom-element.js

```diff
@@ -0,0 +1,27 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class CustomElement extends LitElement {  
+  static get properties(){
+    return {
+      myArray: { type: Array },
+      myBool: { type: Boolean }
+    };
+  }
+  constructor(){
+    super();
+    this.myArray = ['an','array','of','test','data'];
+    this.myBool = true;
+  }
+  render(){
+    return html`
+      <ul>
+        ${this.myArray.map(i => html`<li>${i}</li>`)}
+      </ul>
+      ${this.myBool?
```

Comment:
I would suggest moving the `?` and `:` to the begging of each result for better readability like so:

```js
      ${this.myBool
         ? html`<p>Render some HTML if myBool is true</p>`
         : html`<p>Render some other HTML if myBool is false</p>`
       }
```

Reply:
I like this, will edit samples accordingly if I get time.